pkg/ibmcsidriver/constants.go

```diff
@@ -130,4 +130,4 @@ const (
 var SupportedFS = []string{"nfs"}
 
 // SupportedProfile the supported profile names
-var SupportedProfile = []string{"tier-5iops", "tier-3iops", "tier-10iops", "custom-iops", "dp2"}
```

Comment:
@arahamad please remove all occurrence of constance "CustomProfile" and also the usage of this from controller_helper.go regarding size range checks with iops.

Reply:
done!